### PR TITLE
plugin doc + FAQ fix

### PR DESCRIPTION
**Pull request by DrMegavolt**

### CODE REVIEW

File: docs/plugins.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+layout: doc-section
+title:  "Plugins"
+doc-order: 25.0
+---
+### Plugins (Coming soon)
+
+Plugin is the way Express Gateway can be extended. 
+This feature has a top priority in our Roadmap.
```
Comment (altsang) [inline docs/plugins.md]: Roadmap - please link to waffle board URL

File: docs/plugins.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+### Plugin Structure
+At this point we have all policies, condition etc. built into Express Gateway core. 
+
+So Plugin can have all or some of those:
+* Policies
```
Comment (altsang) [inline docs/plugins.md]: reference link policies, conditions, models

File: docs/plugins.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+* Authentication\Authorization providers
+
+
+
+### Installing a Plugin (Draft)
+
+#### Well known plugin 
+
+In Express Gateway you need to whitelist policies that are allowed to use in pipelines.
+[Whitelist Policies Configuration](./configuration/policies)
```
Comment (altsang) [inline docs/plugins.md]: no mention of it previously , so please remove "Whitelist"

**Comment by DrMegavolt** [comment]
@altsang fixed. the only point is that Conditions are in the pipelines section

**Pull request merged**